linux_os/guide/system/network/network-iptables/iptables_ruleset_modifications/set_iptables_outbound_n_established/tests/rules_established_n_output.pass.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 # platform = multi_platform_all
+# remediation = none
```

Comment:
It's not necessary for the `pass` test scenario scripts, but it's not harmful either. :)